src/cluster.rs

```diff
@@ -619,7 +619,7 @@ impl RoutingInfo {
             b"XGROUP" | b"XINFO" => get_arg(&args, 2).and_then(RoutingInfo::for_key),
             b"XREAD" | b"XREADGROUP" => {
                 let streams_position = args.iter().position(|a| match a {
-                    Value::Data(a) => a == b"STREAMS",
+                    Value::Data(a) => &a.to_ascii_uppercase()[..] == b"STREAMS",
```

Comment:
```suggestion
                    Value::Data(a) => a.eq_ignore_ascii_case(b"STREAMS"),
```
https://doc.rust-lang.org/std/primitive.slice.html#method.eq_ignore_ascii_case